extensions/rich_text_components/components.py

```diff
@@ -121,6 +122,10 @@ class Link(BaseRteComponent):
     """Class for Link component."""
 
 
+class Concept(BaseRteComponent):
+    """Class for Concept component."""
+
+
```

Comment:
Nit: Move this after Collapsible class to keep the alphabetical ordering.

Reply:
Done